src/shared/components/ResourceEditor/CodeEditor.tsx

```diff
@@ -35,7 +35,7 @@ const CodeEditor = forwardRef<codemiror.Editor | undefined, TCodeEditor>(
     return (
       <Spin spinning={busy}>
         <CodeMirror
-          data-testId="code-mirror-editor"
+          data-testid="code-mirror-editor"
```

Comment:
I had to change the attribute name here to get rid of this console error

![Screenshot from 2023-07-04 17-35-59](https://github.com/BlueBrain/nexus-web/assets/11242410/f76a3649-977e-4889-b6c4-2fae756eb43e)